rabbitsmpp/consumer.go

```diff
@@ -69,43 +102,26 @@ func (c *consumer) Consume() (chan Job, chan error, error) {
 				if err != nil {
 					// TODO: Figure out what to do with this failed job
 					log.Printf("failed to unmarshal PDU: %v", err)
-					c.stop("empty body in delivery, stopping consumption")
+					errChan <- err
```

Comment:
yes, errChan is for outside only so the party that reads from that can react on that error if needed. I've changed it to be readOnly since before that we were using to stop the consumers as well.